### PR TITLE
Closes CPPL-55

### PR DESCRIPTION
**Pull request by iodream**
Implemented Login handler and added parser for Authorized handler

Fixed client Login model sending wrong hash data.

### CODE REVIEW

**Review by MaksymTerekhovskyy — APPROVED** [review]

File: src/Client/Models/LoginModel.cpp [hunk 1]
```diff
@@ -1,32 +1,29 @@
...
-    json.insert("login", QString::fromStdString(dto.login));
-    json.insert("password", QString(QCryptographicHash::hash(password
-            , QCryptographicHash::Sha1)));
-
+	json["login"] = dto.login.c_str();
+	auto digest = QString(
+		QCryptographicHash::hash(
+			password, QCryptographicHash::Sha1).toHex());
+	json["password"] = digest.toStdString().c_str();
     return QJsonDocument(json);
```
Comment (yaroslav-kernitskyi) [inline src/Client/Models/LoginModel.cpp:24]: Indentation issue

File: src/Server/Handlers/AuthorizedHandler.cpp [hunk 2]
```diff
@@ -45,13 +54,35 @@ QJsonDocument AuthorizedHandler::DecodeJWTTokenBody(const std::string& token)
 	return QJsonDocument::fromJson(raw_token_body);
 }
 
-bool AuthorizedHandler::CheckAuthorization(const Net::Request& request)
+AuthorizedHandler::JSONParser::DTO AuthorizedHandler::JSONParser::Parse(
+	const QJsonDocument& playload)
```
Comment (yaroslav-kernitskyi) [inline src/Server/Handlers/AuthorizedHandler.cpp:58]: Is `playload` a typo?

File: src/Server/Handlers/Common.h [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 #include "Net/Message.h"
 
+const std::string DB_CONN_STRING = "user=test_user host=127.0.0.1 "
+								   "password=test_pass dbname=SpendAndNote";
```
Comment (yaroslav-kernitskyi) [inline src/Server/Handlers/Common.h:6]: By code convention we should format this thing like
```
const std::string DB_CONN_STRING =
		"user=test_user host=127.0.0.1 "
		"password=test_pass dbname=SpendAndNote";
```

**Review by yaroslav-kernitskyi — APPROVED** [review]
I only found some minor cosmetic issues

File: src/Server/Handlers/AuthorizedHandler.cpp (hunk 2 above)
Comment (iodream) [reply src/Server/Handlers/AuthorizedHandler.cpp]: yes it was. fixed in latest commit

**Pull request merged**